Extensions/PanelSpriteObject/panelspriteruntimeobject-pixi-renderer.ts

```diff
@@ -2,10 +2,14 @@ namespace gdjs {
   import PIXI = GlobalPIXIModule.PIXI;
   class PanelSpriteRuntimeObjectPixiRenderer {
     _object: gdjs.PanelSpriteRuntimeObject;
-    _spritesContainer: any;
-    _centerSprite: any;
-    _borderSprites: any;
-    _alpha: float;
+    // The _wrapperContainer must be considered as the main container of this object
+    // All transformations are applied on this container
+    // See updateOpacity for more info why
+    _wrapperContainer: PIXI.Container;
+    // The _spritesContainer is used to create the sprites and apply cacheAsBitmap only
```

Comment:
Uber nitpicking:
It's I think good practice to end comments with a period.
Also you can use `/** */` comments to that it shows as autocompletion when you hover _wrapperContainer or _spritesContainer.